packages/Molecules/Form/index.vue

```diff
@@ -173,7 +173,10 @@ export default {
 
   .error {
     input,
-    textarea {
+    textarea,
+    .multiselect__tags,
+    .form-control
+    {
```

Comment:
If the SCSS linting worked in .vue files, I think this one would be picked up. { should be on the same line as the last rule.

Reply:
thanks, need to fix that linter!